Server/ReasnAPI/ReasnAPI/Services/TagService.cs

```diff
@@ -119,6 +74,24 @@ public void DeleteTag(int tagId)
         context.SaveChanges();
     }
 
+    public void ForceDeleteTag(int tagId)
```

Comment:
Same as above, think about that we propably don't have tagid in dto

Reply:
changed the logic: we pass the dto then it is checking for name